pkg/analytics/proxy.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2025 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package analytics
+
+const (
+	unsupportedContentTypeEvent = "Unsupported Content Type"
```

Comment:
Nit: Should we specify that is for the deploy proxy or something similar. It is easy we forget about what this event mean in the future